db/migrate/20181003122633_add_canister_id_to_physical_disks.rb

```diff
@@ -0,0 +1,5 @@
+class AddCanisterIdToPhysicalDisks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_disks, :canister_id, :bigint
```

Comment:
You can use `add_reference`  e.g. `add_reference :physical_disks, :canister, :type => :bigint` [ref](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference)

Reply:
@agrare In this case, this affect the way we use `canister_id` on `save_inventory_physical_infra`, right?

Reply:
Not sure what you mean

Reply:
@agrare What I asked is if this affects the way we update the associated IDs for PhysicalDisks on `save_inventory_physical_infra`, more specifically on `h[:canister_id] = h.delete(:canister).try(:[], :id)`, or if `canister_id` column will still exist if we use `add_reference` and nothing else has to change.

Reply:
add_reference will result in a `canister_id` column